### PR TITLE
Attach policy to specified users allowing assumption of the Terraform state read-write role

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request attaches a policy to specified users allowing assumption of the Terraform state read-write role.  It also adds:
- Instructions for operators to help them initially deploy such an environment and grant specific users the ability to redeploy it.
- Instructions for users to help them to set up locally and then redeploy an environment.

See also cisagov/cool-assessment-provisioner-iam#30.

## 💭 Motivation and context ##

It is better to attach the policy in Terraform instead of requiring operations to do so manually in the AWS console.

## 🧪 Testing ##

All automated tests pass.  I deployed these changes to `env6` in our COOL staging environment and verified that they function as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: RedeployingYourOwnEnvironment-Users.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Redeploying Your Own Environment #
+
+In certain cases we specific users to redeploy their own assessment
```
Comment (dav3r) [inline RedeployingYourOwnEnvironment-Users.md]: Add missing word:
```suggestion
In certain cases we allow specific users to redeploy their own assessment
```

**Review by dav3r — APPROVED** [review]
Looks good to me, other than the one missing word that I noted. 👍

**Review by dv4harr10 — APPROVED** [review]

**Comment by dav3r** [comment]
After further reflection, I think that the two markdown files that you added don't belong in this repo.  The info from those files should be passed to our Ops team and users so that it can be stored wherever they keep their various runbooks.

**Comment by jsf9k** [comment]
> After further reflection, I think that the two markdown files that you added don't belong in this repo. The info from those files should be passed to our Ops team and users so that it can be stored wherever they keep their various runbooks.

Those commits have been removed and I have emailed the information to the relevant parties.

**Pull request merged**